docs/beast_development.rst

```diff
@@ -211,8 +207,53 @@ The resulting HTML files are placed in `docs/_build/html` subdirectory, and
 can be viewed in a web browser.
 
 
-Collaborating and Contributing
-==============================
+Submitting a Pull Request
+=========================
+
+Once you have changes that you'd like to contribute back to the project or share
+with collaborators, you can open a pull request. It is a good idea to check with
+the projects or your collaborators which branch of their BEAST repository you
+should send the pull requests.
+
+Pull requests can be submitted at https://github.com/BEAST-Fitting/beast/pulls.
+If you push any commits to your origin repository in a development branch
+(`beast-dev1`), then a "Compare & pull request" button should appear at the
+top of this site. Briefly describe the changes/additions you made in the comments
+section and submit the pull request.
+
+
+Staying up-to-date
+==================
+
+The ``beast`` project's official repository will be updated from time to time
+to accommodate bug fixes, improvements and new features. You can keep your
+fork's `master` repository up-to-date with the following steps:
+
+- Switch to your fork's `master` branch:
+
+  .. code-block:: console
+
+     $ git checkout master
+
+- Fetch the project's up-to-date distribution:
+
+  .. code-block:: console
+
+     $ git fetch upstream
+
+- If there are any differences, then merge the official `master` branch (upstream)
+  with your fork's `master` (you may need to write a commit message):

```

Comment:
How does one check if there are differences?  I usually just run the `git merge` command to find out.

Reply:
That's a good point, although it's possible to fetch upstream and receive no new updates.